modules/custom-status/lib/components/custom-status-editor.js

```diff
@@ -13,7 +13,7 @@ import { useState } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import { useEffect } from 'react';
 
-import ConfirmStatusDeleteModal from './modals/confirm-status-delete-modal';
+import ConfirmDeleteModal from '../../../shared/js/components/modals/confirm-delete-modal';
```

Comment:
It'd be cool if we could get one of those `@shared`-type aliases set up for shared components in JS, e.g.

```suggestion
import ConfirmDeleteModal from '@shared/components/modals/confirm-delete-modal';
```

Reply:
For some reason, I thought that was ts or next.js specific lemme dig into that a bit. That would be a great QOL improvement.

Reply:
Scratch that, I realized that I did try and get this to work using `jsconfig` and `webpack.config.js` but kept running into build problems so I abandoned that for now. Can come back to this later though I 100% agree that this is very useful to have